src/test/scala/abalone/AbaloneFenTest.scala

```diff
@@ -0,0 +1,149 @@
+
+            pieces.get(Pos.E9) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.F9) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.H9) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.I9) must_== Some(Piece(P1, Stone))
+        }
+    }
+
+    "Snakes start position" should {
+        val snakesFen = new format.FEN("PPPPP/5P/6P/1ppppp1P/1p5P1/p1PPPPP1/p6/p5/ppppp 0 0 b 0 0")
+        val pieces = snakesFen.pieces
+
+        "have Black starting the game" in {
+            snakesFen.player must_== Some(P1)
+            snakesFen.value.split(' ').lift(3) must_== Some("b")
+        }
+
+        "have a total of 14 marbles per player" in {
+            pieces.filter(p => p._2.player == P1).size must_== 14
+            pieces.filter(p => p._2.player == P2).size must_== 14
+        }
+    }
+
+    "Atomouche start position" should {
+        val atomoucheFen = new format.FEN("3pP/PpP2p/4P2/p4p1P/P2p1P2p/p1P4P/2p4/P2pPp/pP3 0 0 b 0 0")
+        val pieces = atomoucheFen.pieces
+
+        "have Black starting the game" in {
+            atomoucheFen.player must_== Some(P1)
+            atomoucheFen.value.split(' ').lift(3) must_== Some("b")
+        }
+
+        "have a total of 12 marbles per player" in {
+            pieces.filter(p => p._2.player == P1).size must_== 12
+            pieces.filter(p => p._2.player == P2).size must_== 12
+        }
+
+        "have each marble placed at the expected position" in {
+            pieces.get(Pos.D1) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.B2) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.F2) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.A4) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.F4) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.D5) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.I5) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.B6) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.E7) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.G8) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.I8) must_== Some(Piece(P1, Stone))
+            pieces.get(Pos.E9) must_== Some(Piece(P1, Stone))
+
+            pieces.get(Pos.E1) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.A2) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.C2) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.E3) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.H4) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.A5) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.F5) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.D6) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.I6) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.D8) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.H8) must_== Some(Piece(P2, Stone))
+            pieces.get(Pos.F9) must_== Some(Piece(P2, Stone))
+        }
+    }
+
+
```

Comment:
Please add some fen tests to cover a game partially in progress, and test things like score etc